bundles/org.palladiosimulator.dataflow.confidentiality.analysis/src/org/palladiosimulator/dataflow/confidentiality/analysis/sequence/pcm/PCMDataCharacteristicsCalculator.java

```diff
@@ -7,6 +7,7 @@
 import java.util.stream.Collectors;
 import java.util.Comparator;
 
+import org.apache.log4j.Logger;
```

Comment:
Unused import? At least from what I can see here on github

Reply:
As the duplicate code uses the logger now, this import is now used again